plugins/catalog-backend/src/service/DefaultEntitiesCatalog.ts

```diff
@@ -448,6 +448,13 @@ export class DefaultEntitiesCatalog implements EntitiesCatalog {
       // fetch an extra item to check if there are more items.
       .limit(isFetchingBackwards ? limit : limit + 1);
 
+    if (
+      isQueryEntitiesInitialRequest(request) &&
+      request.offset !== undefined
+    ) {
+      dbQuery.offset(request.offset);
```

Comment:
mmm I don't think this would be so simple, as there are a bunch of operations below which are needed for cursor based pagination and won't be needed anymore for offset pagination.
What's your end goal here? Have you experienced any issue with the current cursor based pagination?

Reply:
So, the end goal here was to: 
1) allow selecting the page size from table
2) allow selecting a page from the dropdown menu
3) support jumping to specific page using URL query parameters (for sharing some view of the catalog)

What comes to the cursor paged pagination, it has its flaws like fast navigation between pages and showing more or fewer items than by default. Additionally, the cursor paged pagination doesn't really support the ordering of items outside the current page. 

I would like to rediscuss this PR if it makes sense.